src/cosl/coordinated_workers/coordinator.py

```diff
@@ -452,6 +455,11 @@ def _s3_config(self) -> Dict[str, Any]:
         s3_config["access_key_id"] = s3_data.pop("access-key")
         s3_config["secret_access_key"] = s3_data.pop("secret-key")
         s3_config["bucket_name"] = s3_data.pop("bucket")
+        ca_chain = s3_data.get("tls-ca-chain")
+        if ca_chain:
+            # put the cacert to disk
+            Path(S3_CA_CERT_PATH).write_text(ca_chain[0])  # TODO: is any cert in the chain good?
```

Comment:
I dont think it's returned as a list. Its just a concatenated string, so we can dump all in one file and we'll be fine.

Reply:
it is a list, I checked. it gets json-loaded by the lib

Reply:
Right.
Well in that case, picking the 1st cert can be problematic, as it can contain the server cert not even a CA
Can we maybe join all with a `\n\n` like [this](https://github.com/canonical/observability-libs/blob/main/lib/charms/observability_libs/v1/cert_handler.py#L598)?

Reply:
okay, I guess we'll need to try this out somehow